src/encoded/types/target.py

```diff
@@ -36,13 +37,13 @@ def unique_keys(self, properties):
         "type": "string",
         "linkTo": "Organism"
     })
-    def organism(self, properties=None):
+    def organism(self, properties=None, return_uuid=False):
         if properties is None:
             properties = self.upgrade_properties()
+        root = find_root(self)
         if 'target_organism' in properties:
-            return properties['target_organism']
+            organism_uuid = properties['target_organism']
```

Comment:
Do you know if the target_organism is a uuid here? Was assuming it would be @id since it is a specified property. Passing the @id to root.get_by_uuid would return blank result. We should add test for this.

Reply:
Two unit tests added for posting non-gene targets with target_organism being @id and uuid respectively. Both cases have the organism property calculated correctly.